blocked-edges/4.12.17-SpuriousCCMOUpgradeableCond.yaml

```diff
@@ -0,0 +1,10 @@
+to: 4.12.17
+from: .*
+fixedIn: 4.12.18
```

Comment:
It's not technically fully fixed but the bot would nag us if we did not have this